site/en/_posts/2020-02-19-dhd.md

```diff
@@ -8,4 +8,4 @@ author: Elisabeth Engl
 
 The OCR-D funding initiative will be represented with several presentations at the DHd 2020, which will take place in Paderborn from 2-6 March.
 
-In V15, on March 5, the OCR-D coordination project will discuss the results and perspectives of the funding initiative on the basis of four theses on full text transformation. In the same section, the colleagues of the module project for the development of a model repository and automatic font recognition for OCR-D will report on their findings on the use of fracture in German-language books. For further information on the presentations see [the DHd programm](https://www.conftool.com/dhd2020/index.php?page=browseSessions&form_session=124&presentations=show).
+In V15, on March 5, the OCR-D coordination project will discuss the results and perspectives of the funding initiative on the basis of four theses on full text transformation. In the same section, the colleagues of the module project for the development of a model repository and automatic font recognition for OCR-D will report on their findings on the use of fracture in German-language books. For further information on the presentations see [the DHd program](https://www.conftool.com/dhd2020/index.php?page=browseSessions&form_session=124&presentations=show).

```

Comment:
programm -> program

Reply:
feel free to correct directly in the PR, thanks!

Reply:
My comment was just to indicate what was changed in that lengthy text.

Reply:
Right, sry, I had only seen the e-mail.